example/src/main/scala/example/FileStreaming.scala

```diff
@@ -5,18 +5,31 @@ import zhttp.service.Server
 import zio.stream.ZStream
 import zio.{App, ExitCode, URIO}
 
-import java.nio.file.Paths
+import java.io.File
+import java.nio.charset.StandardCharsets
+import java.nio.file.{Files, Paths}
 
 object FileStreaming extends App {
   // Read the file as ZStream
-  val content = HttpData.fromStream {
+  val content          = HttpData.fromStream {
     ZStream.fromFile(Paths.get("README.md"))
   }
+  val filePathAsString = File.createTempFile("example/blahFile", ".avi").getAbsolutePath
+
+  def testFile = {
+    val tempFileName = File.createTempFile("example/blahTextFile", ".txt").getAbsolutePath
+    Files.write(
+      Paths.get(tempFileName),
+      "This is a test file created only for testing.\n".getBytes(StandardCharsets.UTF_8),
+    )
+  }
```

Comment:
Can add a file to resources and serve it directly instead of creating one in the example.